packages/terra-table/src/TableSubheader.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import 'terra-base/lib/baseStyles';
+
+const propTypes = {
+  /**
+   * Content to be displayed in the subheader row
+   */
+  content: PropTypes.node.isRequired,
+
+  /**
+   * Number of columns the subheader must span
+   */
+  count: PropTypes.number,
```

Comment:
It seems that this component is used internally by TableRows, but would we want to update this prop name to colSpan to be more descriptive?